spec/system/planning_applications/publicity/press_notice_spec.rb

```diff
@@ -7,7 +7,7 @@
   let!(:assessor) { create(:user, :assessor, local_authority:) }
 
   let!(:planning_application) do
-    create(:planning_application, :prior_approval, local_authority:)
+    create(:planning_application, :prior_approval, local_authority:, postcode: "")
```

Comment:
I'm not entirely sure why this was necessary, but changing the `travel_to`s in this spec caused webmock to start complaining about there not being any stubs for the mapit api. A blank postcode means no requests will be made to the mapit api so avoids the issue.

Reply:
That sounds like the behaviour I had on Petitions with that Chrome bug - requests were still happening after the end of the spec and webmock had been reset.